poc/tenant-controller/pkg/controllers/tenants/controller.go

```diff
@@ -63,7 +70,21 @@ func (c *Controller) Run(ctx context.Context) error {
 func (c *Controller) createTenant(obj *tenantsapi.Tenant) {
 	glog.Info("createTenant: %#v", obj)
 	syncRBACForTenantCRD(obj)
-	// TODO create namespace
+
+	// TODO Add later ... sanity checks to ensure namespaces being requested are valid and not already assigned to another tenant
+	c.tenantsList = make(map[string]tenantsapi.Tenant)
+	c.tenantsList[obj.ObjectMeta.Name] = *obj
+
+	namespace := corev1.Namespace{}
```

Comment:
It's not a good idea to share a single object in multiple for loop iterations below.

Reply:
Okay, will change it per comment above.